NinchatSDKSwift/Implementations/Utilities/VideoThumbnailManager.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) 17.3.2020 Somia Reality Oy. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+
+import AVFoundation
+import UIKit
+
+struct VideoThumbnailManager {
```

Comment:
What video thumbnails are this regarding? Ninchat attachments don't have thumbnails for video files.

Reply:
We show a thumbnail for video attachments on the iOS which is extracted from the video locally. It’s very helpful for users, especially because videos are played within the application, not on  an external browser.

Reply:
Ok